clients/crystal/github.com/vladfaust/mini_redis.json

```
@@ -1,8 +1,8 @@
 {
     "name": "mini_redis",
     "description": "A light-weight low-level Redis client for Crystal",
-    "homepage": "http://github.vladfaust.com/mini_redis/",
+    "homepage": "http://github.com/vladfaust/mini_redis/",
```

maybe https is better? (not that important)
```suggestion
    "homepage": "https://github.com/vladfaust/mini_redis/",
```